README.md

```diff
@@ -79,6 +79,14 @@ Run `make archive` from the root directory to generate `.xcframeworks` for each
 
 Additional documentation for usage and SDK architecture can be found under the [Documentation](Documentation/README.md) directory.
 
+## Related Projects
+
+[AEP SDK Sample App for iOS](https://github.com/adobe/aepsdk-sample-app-ios) contains iOS sample apps for the AEP SDK. Examples are provided for both Objective-c and Swift implementation.
+
+[AEP SDK Sample Extension for iOS](https://github.com/adobe/aepsdk-sample-extension-ios) contains a sample implementation of an iOS extension for the AEP SDK. Example implementations are provided for both Objective-c and Swift.
+
+[AEP SDK Compatibility for iOS](https://github.com/adobe/aepsdk-compatibility-ios) contains the code required to upgrade from the existing ACP SDK to the latest AEP SDK without introducing breaking changes so you can enjoy the benefits of the newest AEP SDK without needing to reimplement the SDK.
```

Comment:
"contains code required to migrate from the existing ACP SDK to this AEP SDK without making implementation changes. While updating your implementation is recommended, this option allows you to upgrade to the AEP SDK with minimal effort."

thoughts?